esp-hal/MIGRATING-0.22.md

```diff
@@ -140,6 +151,24 @@ Timer group timers have been type erased.
 + timg::Timer
 ```
 
+The usage of `esp_alloc::psram_allocator!` remains unchanged.
```

Comment:
something weird happend to the file - this seems to be moved here from the PSRAM chapter and the whole `embedded-hal 0.2.* is not supported anymore` chapter disappeared